service/src/main/java/org/apache/griffin/core/job/SparkSubmitJob.java

```diff
@@ -98,7 +100,12 @@ public void execute(JobExecutionContext context) {
                 updateJobInstanceState(context);
                 return;
             }
-            saveJobInstance(jd);
+            if(livyTaskSubmitHelper.isNeedLivyQueue()){
+                //livy batch limit
+                livyTaskSubmitHelper.addTaskToWaitingQueue(livyConfMap);
+            }else{
+                saveJobInstance(jd);
+            }
```

Comment:
As you modify the job execution process, you need to update relevant tests in SparkSubmitJobTest class to make travis build pass.

Reply:
> As you modify the job execution process, you need to update relevant tests in SparkSubmitJobTest class to make travis build pass.

Thanks. Add mock bean in test and build success.